juriscraper/scotus/scotus_email.py

```diff
@@ -16,43 +17,86 @@
 from juriscraper.lib.string_utils import clean_string, harmonize
 
 
+class EmailType(Enum):
+    INVALID = auto()
+    CONFIRMATION = auto()
+    DOCKET_ENTRY = auto()
```

Comment:
I suggest using literals here so we can leverage them when rendering the `email_type`:

```suggestion
    INVALID = "invalid"
    CONFIRMATION = "confirmation"
    DOCKET_ENTRY = "docket_entry"
```